app/views/shared/samples/spreadsheet_imports/_dialog.html.erb

```diff
@@ -0,0 +1,113 @@
+<%= viral_dialog(open: open, size: :large, closable: closable) do |dialog| %>
+  <% dialog.with_header(title: t(".title")) %>
+  <%= turbo_stream_from @broadcast_target %>
+  <%= turbo_frame_tag "import_spreadsheet_dialog_content" do %>
+
+    <%= turbo_frame_tag "import_spreadsheet_dialog_alert" %>
+
+    <div
+      data-controller="spreadsheet-import"
+      <% if @namespace.type == 'Group' %>
```

Comment:
Can change this to `@namespace.group_namespace?`

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/984/commits/1c30d7b0e5a3d125104197aab4e8b2066f557f74